app/src/main/kotlin/com/saltedge/authenticator/tools/AppTools.kt

```diff
@@ -105,3 +105,38 @@ val FragmentActivity.authenticatorApp: AuthenticatorApplication?
  */
 val Fragment.authenticatorApp: AuthenticatorApplication?
     get() = this.activity?.authenticatorApp
+
+/**
+ * Check if system sdk version is 28 or greater
+ *
+ * @return boolean, true if version sdk is greater than or equal to VERSION_CODES.P (SDK28)
+ */
+val buildVersion28OrGreater: Boolean = Build.VERSION.SDK_INT >= Build.VERSION_CODES.P
```

Comment:
Let's switch places `Version` and number of version, on smth like this:
`build28VersionOrGreater`, because I see 280